src/main/java/io/milvus/v2/client/ConnectConfig.java

```diff
@@ -57,6 +58,8 @@ public class ConnectConfig {
     @Builder.Default
     private long idleTimeoutMs = TimeUnit.MILLISECONDS.convert(24, TimeUnit.HOURS);
 
+    private SslContext sslContext;
```

Comment:
Instead of injecting the Netty `SslContext`, it seems worth injecting the `javax.net.ssl.SSLContext` and then converting it to the Netty `SslContext` instead of the `ClientUtils` class. That way it avoids introducing the shaded Netty class reference into the public `ConnectConfig` class.

Reply:
For reference, the [JdkSslContext](https://netty.io/4.1/api/io/netty/handler/ssl/JdkSslContext.html) provides the translation from the Java `SSLContext` to the Netty version.

Reply:
Addressed suggestion above, changed to SSLContext on ConnectConfig and added conversion in the ClientUtils method